defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 zookeeper_version: 3.4.11
```

Comment:
@cifaz you haved bumped in README.md but not in main.yml :)

Reply:
And any else?